dasharo-compatibility/sata-detect.robot

```diff
@@ -0,0 +1,87 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+# Library    ../lib/QemuMonitor.py    /tmp/qmp-socket
+# Required setup keywords:
+#    Prepare Test Suite - generic setup keyword for all tests
+# Required teardown keywords:
+#    Log Out And Close Connection - generic setup keyword for all tests,
+#    closes all connections to DUT and PiKVM
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Skip If    not ${ESP_SCANNING_SUPPORT}    ESP scanning tests not supported
+...                     AND
+...                     Prepare Required Files For Qemu
+...                     AND
+...                     Prepare EFI Partition With System Files
+...                     AND
+...                     Restore Initial DUT Connection Method
+Suite Teardown      Run Keywords
+...                     Clear Out EFI Partition    AND
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+SAT001.001 ESP Scan should contain SATA if it is present
+    [Documentation]    This test aims to verify that SATA shows up in the
+    ...    boot menu
+    Sleep    1
```

Comment:
what is the purpose of this sleep?

Reply:
@macpijan I forgot to remove it before pushing, my bad. Fixed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/290/commits/1f840797226c247d3ffda3121394867a1fbe0ab0#diff-8229e9b186f72246bc5fa0c7547eb2299b6dfb3bc416446fab9430a947eca420L40)